src/components/NoteDisplay/NonDeleteMode.js

```diff
@@ -0,0 +1,49 @@
+import React, { useContext } from "react";
+import { useHistory } from 'react-router-dom';
+import Icon from "../shared/Icon";
+import Button from "../shared/Button";
+import iconDir from "../icon.svg";
+import globalTypes from "../../GlobalTypes";
+import { AppContext } from "../../contexts/AppContext";
+import { ThemeContext } from "../../contexts/ThemeContext";
```

Comment:
What are your thoughts in using path aliases like `@/context/AppContext'` for contexts as to avoid this kind of relative imports.